### ISSUE TITLE
Highlight add now requires Manage Messages permission

### ISSUE THREAD

**Issue by zinabat** [CarlGroth/Carl-Bot#49]
The highlight command used to work for all regular members. It now requires the Manage Messages permission, which seems incorrect. This has broken the functionality for users who wish to use the highlights feature except for moderators/staff members.

## Example
![Screen Shot 2022-05-01 at 2 20 49 PM](https://user-images.githubusercontent.com/2412740/166159147-3e769dd4-8e96-4fd3-bdc4-51fde34b7798.png)

**Comment by devargon** [comment]
This repo hasn't been maintained in 2 years lol

**Comment by CarlGroth** [comment]
Yeah it's ridiculous. I don't run the bot anymore but this change is just because the people who took over couldn't scale the feature (hint: it's called aho-corasick)
